test/driver/exception_handling/run.t

```diff
@@ -112,12 +157,48 @@ and the whole AST is prepended with an error extension node.
 
   $ echo "type a = int" > impl.ml
   $ echo "type b = int [@@deriving deriver_raised_exception]" >> impl.ml
+  $ echo "type b = int [@@deriving deriver_raised_exception2]" >> impl.ml
   $ ./deriver.exe -embed-errors impl.ml
   Fatal error: exception Failure("A raised exception")
   [2]
 
+ In the case of Constant types
+
+  $ echo "let x = 2g + 3g" > impl.ml
+  $ echo "let x = 2g + 3g" >> impl.ml
+
+ When embed-errors is not passed 
+  $ ./constant_type.exe impl.ml
+  File "impl.ml", line 1, characters 8-10:
+  Error: rewriter 2 failed
+  [1]
+
+ When embed-errors is not passed
+  $ ./constant_type.exe -embed-errors impl.ml
+  [%%ocaml.error "rewriter 2 failed"]
+  [%%ocaml.error "rewriter 3 failed"]
+  [%%ocaml.error "rewriter 2 failed"]
+  [%%ocaml.error "rewriter 3 failed"]
```

Comment:
Maybe the "constant rewriting" error message could be improved! It is not trivial to link the list of error messages to the original code...